CRM/Contribute/BAO/Query.php

```diff
@@ -975,7 +941,7 @@ public static function defaultReturnProperties($mode, $includeCustomFields = TRU
         // actually
         'is_pay_later' => 1,
         // required
-        'contribution_status' => 1,
+        // 'contribution_status' => 1,
```

Comment:
well once the pseudo attributes work correctly they automatically add contribution_status when contribution_status_id is requested  - so it's redundant